public/tier1/KeyValues.h

```diff
@@ -139,24 +160,26 @@ class KeyValues
 
 	// Find a keyValue, create it if it is not found.
 	// Set bCreate to true to create the key if it doesn't already exist (which ensures a valid pointer will be returned)
-	KeyValues *FindKey(const char *keyName, bool bCreate = false);
-	KeyValues *FindKey(HKeySymbol keySymbol) const;
+	KeyValues *FindKey( const char *keyName, bool bCreate = false );
+	KeyValues *FindKey( const char *keyName ) const;

```

Comment:
These 2 definitions conflict each other. Calling `FindKey( "X" )` would be ambiguous.